onnxruntime/test/onnx/testenv.cc

```diff
@@ -12,18 +12,9 @@
 
 using onnxruntime::Status;
 
-static std::unique_ptr<onnxruntime::concurrency::ThreadPool> default_pool;
-static std::once_flag default_pool_init;
-
-PThreadPool TestEnv::GetDefaultThreadPool(onnxruntime::Env& env) {
-  std::call_once(default_pool_init, [&env] {
-    using namespace onnxruntime::concurrency;
+std::unique_ptr<OrtThreadPool> TestEnv::CreateThreadPool(onnxruntime::Env& env) {
     int core_num = env.GetNumPhysicalCpuCores();
-
-    onnxruntime::ThreadOptions t_opts;
-    default_pool = std::make_unique<ThreadPool>(&env, t_opts, ORT_TSTR("onnx_runner_tp"), core_num, false);
-  });
-  return default_pool.get();
+    return std::make_unique<OrtThreadPool>(&env, onnxruntime::ThreadOptions{}, ORT_TSTR("onnx_runner_tp"), core_num, false);
```

Comment:
I wonder why this was called within call_once before. Hopefully this is not getting called in a loop somewhere. Can you confirm this?
Moreover, we might want to rename this function to CreateThreadPool to reflect its true job.

Reply:
The function already renamed to "CreateThreadPool".
Let me double check very of its occurrence to double confirm.

Reply:
https://github.com/microsoft/onnxruntime/blob/a81faee41ef2344de448caecb0f42a34fdc9ead7/onnxruntime/test/providers/cpu/model_tests.cc#L745
Should be the only other place calling the function.